lang/en.json

```diff
@@ -840,5 +844,9 @@
     "lpaWithdrawn": "LPA withdrawn",
     "youHaveWithdrawnLpaNumber": "You have withdrawn LPA number <span class=\"govuk-!-font-weight-bold\">{{.UID}}</span>.",
     "opgWillNowContactAnyoneWhoHasAlreadyBeenContacted": "OPG will now contact anyone who has already been contacted in relation to your LPA and notify them of your decision to withdraw.",
-    "withdrawn": "Withdrawn"
+    "withdrawn": "Withdrawn",
+    "yourFilesAreUploading": "Your files are uploading",
```

Comment:
![image](https://github.com/ministryofjustice/opg-modernising-lpa/assets/146390/37c9780a-544c-48ae-8fde-ef1d5ae638be)

the wording on this page is now weird, "Your files are uploading" even though they are in a section called "Uploaded files"

Reply:
Missed this - it will make more sense to have the modal appear when clicking upload files rather than continue.